TX.h

```diff
@@ -538,15 +538,18 @@ static inline void processSBUS(uint8_t c)
     if ((frameIndex == 23) && (c == SBUS_TAIL)) {
       uint8_t set;
       for (set = 0; set < 2; set++) {
-        PPM[(set<<3)] = ppmWork.sbus.ch[set].ch0 >> 1;
-        PPM[(set<<3)+1] = ppmWork.sbus.ch[set].ch1 >> 1;
-        PPM[(set<<3)+2] = ppmWork.sbus.ch[set].ch2 >> 1;
-        PPM[(set<<3)+3] = ppmWork.sbus.ch[set].ch3 >> 1;
-        PPM[(set<<3)+4] = ppmWork.sbus.ch[set].ch4 >> 1;
-        PPM[(set<<3)+5] = ppmWork.sbus.ch[set].ch5 >> 1;
-        PPM[(set<<3)+6] = ppmWork.sbus.ch[set].ch6 >> 1;
-        PPM[(set<<3)+7] = ppmWork.sbus.ch[set].ch7 >> 1;
+        PPM[(set<<3)] = (((ppmWork.sbus.ch[set].ch0 >> 1) * 5) >> 2);
+        PPM[(set<<3)+1] = (((ppmWork.sbus.ch[set].ch1 >> 1) * 5) >> 2);
+        PPM[(set<<3)+2] = (((ppmWork.sbus.ch[set].ch2 >> 1) * 5) >> 2);
+        PPM[(set<<3)+3] = (((ppmWork.sbus.ch[set].ch3 >> 1) * 5) >> 2);
+        PPM[(set<<3)+4] = (((ppmWork.sbus.ch[set].ch4 >> 1) * 5) >> 2);
+        PPM[(set<<3)+5] = (((ppmWork.sbus.ch[set].ch5 >> 1) * 5) >> 2);
+        PPM[(set<<3)+6] = (((ppmWork.sbus.ch[set].ch6 >> 1) * 5) >> 2);
+        PPM[(set<<3)+7] = (((ppmWork.sbus.ch[set].ch7 >> 1) * 5) >> 2);
       }
+      for(uint8_t i=0; i<16; i++){
+       PPM[i]=(PPM[i]<=108)?0:(PPM[i]-108);
```

Comment:
The math is a bit confusing, but it seems to work out. Based on raw data & info I found online, raw SBUS values can range from 172-1812, which corresponds to a PPM signal range of ~988-2012us (on a taranis w/openTX).  Without this fix, the values were somewhat off (ex. center stick value of '1500us' on radio was showing up as ~1484us on the openLRS side). The scaling formula fixes this.

The ['servoBits2Us()'](https://github.com/openLRSng/openLRSng/blob/master/common.h#L125) function takes care of adjusting the value ranges for the PPM/PWM output to the 12-1012 range. [Check this comment](https://github.com/openLRSng/openLRSng/blob/master/common.h#L101) for the output range translation.

However, now that you mention it, I probably should have added an upper bound/limit test to keep the values within the 0-1024 range. Otherwise, it can screw up the 10-bit packing for the RF transmit packets.

Please feel free to double-check the math & test if possible. Let me know if you still find something off or broken...